noir-projects/aztec-nr/value-note/src/value_note.nr

```diff
@@ -18,10 +18,7 @@ use dep::aztec::{
 pub(crate) global VALUE_NOTE_LEN: u32 = 3; // 3 plus a header.
 
 // docs:start:value-note-def
-// ValueNote is used as fn parameter in the Claim contract, so it has to implement the Serialize trait.
-// It is important that the order of these annotations is preserved so that derive(Serialize) runs AFTER the note macro, which injects the note header.
 #[note]
-#[derive(Serialize)]
```

Comment:
This was no longer needed because now we use UintNote in that contract. Forgot to do this change in that PR.